syft/frameworks/torch/hook/hook.py

```diff
@@ -145,8 +142,11 @@ def __init__(
         self._hook_native_tensor(torch.Tensor, TorchTensor)
 
         if dependency_check.crypten_available:
-            self.to_auto_overload[crypten.mpc.MPCTensor] = ["get_plain_text"]
-            self._hook_syft_placeholder_methods(crypten.mpc.MPCTensor, PlaceHolder)
+            from syft.frameworks.crypten.hook.hook import crypten_to_auto_overload
+
+            for component, methods in crypten_to_auto_overload.items():
+                self.to_auto_overload[component] = methods
+                self._hook_syft_placeholder_methods(component, PlaceHolder)
```

Comment:
maybe rename
component -> crypten_class
methods -> method_names
for optimal readibility?

Reply:
Done